pkgs/applications/window-managers/picom/picom-next.nix

```
@@ -0,0 +1,13 @@
+{ picom, lib, fetchFromGitHub }:
+
+picom.overrideAttrs (oldAttrs: rec {
+  pname = "picom-next";
+  version = "unstable-2021-11-01";
```

```suggestion
  version = "unstable-2021-10-31";
```

Date must be the date of fetched commit (not the date of the PR).

---

Sorry im in a different timezone so i got the date wrong